hypertrace-graphql-entity-schema/src/main/java/org/hypertrace/graphql/entity/request/EdgeSetGroupRequest.java

```diff
@@ -13,4 +14,8 @@ public interface EdgeSetGroupRequest {
   Single<EntityRequest> buildNeighborRequest(String entityType, Collection<String> neighborIds);
 
   Map<String, EdgeSetRequest> edgeSetRequests();
+
+  Collection<AttributeRequest> getAllAttributeRequests();
```

Comment:
We're taking a step backwards moving these up to the request - remember each edge set has different selections, and that's why we broke them down.

Reply:
but we need them right its getting duplicated otherwise? a util maybe?

Reply:
Nope - the edge fetcher shouldn't need them either - https://github.com/hypertrace/hypertrace-graphql/pull/203#discussion_r1408050992